package.json

```diff
@@ -275,7 +277,7 @@
 		"build:plugin-zip": "bash ./bin/build-plugin-zip.sh",
 		"clean:package-types": "tsc --build --clean && rimraf \"./packages/*/build-types\"",
 		"clean:packages": "rimraf \"./packages/*/@(build|build-module|build-style)\"",
-		"dev": "cross-env NODE_ENV=development npm run build:packages && concurrently \"wp-scripts start\" \"npm run dev:packages\"",
+		"dev": "cross-env NODE_ENV=development npm run build:packages && concurrently \"wp-scripts serve\" \"npm run dev:packages\"",
```

Comment:
What are the limitation of `wp-scripts start --hot` that force introducing a new script? I would be more in favor of always using the dev server for `start` by default even when not using hot module replacement.

Reply:
We want to use our own manual HMR setup so we don't want to pass `--hot` to webpack. `wp-scripts start --hot` seems to always pass that to webpack which then override the `hot: false` in the config.

I agree using `start` will be ideal but `serve` seems reasonable too, given that the official webpack command uses [`serve`](https://webpack.js.org/configuration/dev-server/#usage-via-cli) to start a dev server too.